data/gnome/check-svg.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+# vim: set expandtab shiftwidth=4 tabstop=4:
+
+import sys
+from lxml import etree
+import logging
+
+ns = {'svg': 'http://www.w3.org/2000/svg'}
+
+logging.basicConfig(level=logging.DEBUG)
+logger = None
+success = True
+
+
+def check_size(root):
+    width = float(root.attrib['width'])
+    height = float(root.attrib['height'])
+    if width < 400 or width > 500:
```

Comment:
Just FYI: Python supports `400 < width < 500` as well. Just a matter of preference really, but this can alternatively be `if not 400 < width < 500`

Reply:
ooh, nice. fixed locally, will be in the force-push